### PR TITLE
Added us002 API.

### PR DESCRIPTION
**Pull request by BlaNaoZ**

### CODE REVIEW

File: src/main/java/org/kainos/ea/models/JobSpecificationResponse.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    public String getLink() {
+        return link;
+    }
+
+    public void setLink(String link) {
+        this.link = link;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```
Comment (shaunganley) [inline src/main/java/org/kainos/ea/models/JobSpecificationResponse.java:52]: do you need these overridden methods?

File: src/main/java/org/kainos/ea/resources/WebService.java [hunk 2]
```diff
@@ -33,4 +37,25 @@ public Response getJobRoles() throws SQLException, DatabaseConnectionException {
...
+
+    @GET
+    @Path("/job-specification/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getJobSpecification( @PathParam("id") int id ) throws SQLException, DatabaseConnectionException {
+
+        try {
+            JobSpecificationResponse jobSpecification = jobsService.getJobSpecification(id);
+
+            if(jobSpecification.getTitle() == null) {
```
Comment (shaunganley) [inline src/main/java/org/kainos/ea/resources/WebService.java]: can you move this logic to the service and unit test (you'll need to throw an exception which you can catch below)

**Review by shaunganley — CHANGES_REQUESTED** [review]
great work, 2 small comments

File: src/main/java/org/kainos/ea/models/JobSpecificationResponse.java (hunk 1 above)
Comment (BlaNaoZ) [reply src/main/java/org/kainos/ea/models/JobSpecificationResponse.java:52]: I'm using the equals method for testing (asserting whether two responses have the same data). The hashCode one is indeed redundant, so I'll remove it.

Edit: Removed the hashCode method.

File: src/main/java/org/kainos/ea/resources/WebService.java (hunk 2 above)
Comment (BlaNaoZ) [reply src/main/java/org/kainos/ea/resources/WebService.java]: I Implemented this.

**Review by shaunganley — APPROVED** [review]

**Review by laura-puckoriute — APPROVED** [review]

**Pull request merged**